src/foam/dao/MaterializedDAO.js

```diff
@@ -97,11 +97,10 @@ foam.CLASS({
       type: 'foam.nanos.auth.Authorizer',
       name: 'authorizer',
       javaFactory: `
-      String sourceClass = getSourceDAO().getOf().getObjClass().getSimpleName().toLowerCase();
-      if ( foam.nanos.auth.Authorizable.class.isAssignableFrom(getOf().getObjClass()) ) {
-        return new foam.nanos.auth.AuthorizableAuthorizer(sourceClass);
+      if ( foam.nanos.auth.Authorizable.class.isAssignableFrom(getSourceDAO().getOf().getObjClass()) ) {
```

Comment:
@noodlemoodle `getSourceDAO().getOf().getObjClass()` may not be the same as `getOf().getObjClass()`, the `of` of the mat dao could be different from the `of` of the source dao.

The AuthorizableAuthorizer should only apply when the mat dao `of` implements Authorizable.